pkgs/development/compilers/elm/packages/node-packages.json

```diff
@@ -9,5 +9,7 @@
     "elm-verify-examples",
     "elm-xref",
     "create-elm-app",
-    "elm-optimize-level-2"
+    "elm-optimize-level-2",
```

Comment:
this package was added in https://github.com/NixOS/nixpkgs/pull/96594

Reply:
What exactly are you getting at here?

Reply:
just cleaning up old PRs. This one went under the carpet since no committer spotted it. Since then elm-optimize-level-2 was added. This adds elm-git-install https://github.com/NixOS/nixpkgs/pull/150458. That means `elm-install` is the only package from this PR which is not merged. I wonder - do you still want to include it in nixpkgs?

Reply:
> do you still want to include it in nixpkgs?

I don't see myself writing elm any time soon, so I'm kinda indifferent about it at this point. But if I don't have to do much to get it in, then yeah, why not.